ui/test/nuxeo-html-editor.test.js

```diff
@@ -0,0 +1,55 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { fixture, html } from '@nuxeo/nuxeo-elements/test/test-helpers';
+import '../widgets/nuxeo-html-editor.js';
+
+/* eslint-disable no-unused-expressions */
+suite('<nuxeo-html-editor>', () => {
+  test('should have empty default value and display the placeholder', async () => {
+    const editor = await fixture(
+      html`
+        <nuxeo-html-editor></nuxeo-html-editor>
+      `,
+    );
+    expect(editor.value).to.be.undefined;
+    expect(editor._editor.root.dataset.placeholder).to.equal('Type here...');
+  });
+
+  test('should sync the html value', async () => {
+    const value = '<p>Hello</p>';
+    const editor = await fixture(
+      html`
+        <nuxeo-html-editor value="${value}"></nuxeo-html-editor>
+      `,
+    );
+    expect(editor.value).to.equal(value);
+    await new Promise((resolve) => {
+      editor.addEventListener('value-changed', () => resolve());
+      editor._insertText(' world!');
+    });
+    expect(editor.value).to.equal(`<p>Hello world!</p>`);
+  });
+
+  test('should hide the toolbar when readonly', async () => {
```

Comment:
It seems that this test is not doing its job, correct? I believe we changed the flag name on the element itself